packages/sdk/src/contracts/ERC1271ContractFacade.ts

```diff
@@ -36,7 +36,7 @@ export class ERC1271ContractFacade {
     }
 
     async isValidSignature(contractAddress: EthereumAddress, payload: Uint8Array, signature: Uint8Array): Promise<boolean> {
-        const recoveredSignerUserId = toEthereumAddress(binaryToHex(recoverSignerUserId(signature, payload), true))
+        const recoveredSignerUserId = toUserId(recoverSignerUserId(signature, payload))
```

Comment:
I imagine that according to https://eips.ethereum.org/EIPS/eip-1271 the "recovered id" will be an address, so why change this?

Reply:
Removed the `getAddress()` method so that we don't have two different names for the same thing. 

Typically the method is used like this:
```ts
await stream.grantPermissions({
    permissions: [StreamPermission.PUBLISH],
    userId: await publisher.getUserId()
})
```

-> I.e. when there is a `userId` argument in the API, the user should put something like `client.getUserId()` to it. It it would be like this, it would be less straightforward:
```ts
await stream.grantPermissions({
    permissions: [StreamPermission.PUBLISH],
    userId: await publisher.getAddress()
})
```